doc/whats_new/v0.20.rst

```diff
@@ -78,6 +78,14 @@ Model evaluation
 - Added the :func:`metrics.balanced_accuracy_score` metric and a corresponding
   ``'balanced_accuracy'`` scorer for binary classification.
   :issue:`8066` by :user:`xyguo` and :user:`Aman Dalmia <dalmia>`.
+
+Misc
+
+- A new configuration parameter, ``working_memory`` was added to control memory
+  consumption limits in chunked operations, such as the new
+  :func:`metrics.pairwise_distances_chunked`.  See :ref:`working_memory`.
```

Comment:
I think a reference to the User Guide is most relevant in what's new.

I can add a glossary entry, though I'm not sure how it will help beyond the user guide and the `config_context` docstring.

Reply:
Fair enough, I just wonder what is the goal of the syntax `:ref:`, which [does not render a link](https://23284-843222-gh.circle-artifacts.com/0/doc/whats_new/v0.20.html#new-features).

Did you mean `:func:set_config`? Or maybe you need a label in `doc/modules/computational_performance.rst`?

Reply:
The latter. A glossary reference would be `:term:`, not `:ref:` which references sections.